### PR TITLE
Create a shared cover crop council logo displayer #4

### LINKED ISSUE #4: Create a shared cover crop council logo displayer
**What's wrong**
- We want to make the Cover Crop Council Logo reusable between all of the different applications. 

**What done looks like**
- We need to create a Cover Crop Council Logo component that can be reused in all of our applications.  The Cover Crop Council Logo will need to have any information it holds passed into it so it can be reused and replace all other places the logo will be displayed .  

**Where the changes need to be made**
- anywhere you find the cover crop council logo 

please name this
PSACoverCropCouncil.jsx

### PR DESCRIPTION
**Pull request by ravighevariya10**
Created a shared CoverCropCouncil logo displayer

### CODE REVIEW

File: src/logo-displayer/PSACoverCropCouncil.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useRef } from 'react';
+
+const PSACoverCropCouncil = ({
+  councilShorthandRedux,
```
Comment (YazidZidane) [inline src/logo-displayer/PSACoverCropCouncil.jsx]: Can you rename this to `council`?

File: src/logo-displayer/PSACoverCropCouncil.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  return (
+    <img
+      ref={logoRef}
+      alt={alt}
+      style={style}
+    />
+  );
+};
+
+export default PSACoverCropCouncil;
```
Comment (YazidZidane) [inline src/logo-displayer/PSACoverCropCouncil.jsx]: Can you also add descriptions to the props using `propTypes`? You can refer to other components for that

**Comment by YazidZidane** [comment]
Can you also rename the component to `PSALogoDisplayer` or something like that to make it more meaningful? And also add the component to `src/index.js`

File: src/logo-displayer/PSACoverCropCouncil.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      alt={alt}
+      style={style}
+    />
+  );
+};
+
+PSALogoDisplayer.propTypes = {
+  council: PropTypes.string,
+  alt: PropTypes.string,
+  style: PropTypes.object
```
Comment (YazidZidane) [inline src/logo-displayer/PSACoverCropCouncil.jsx:76]: Can you also add descriptions to each props in JSdoc format just like other components?
```
/**
 * add comments here
 */
```

Comment (ravighevariya10) [reply src/logo-displayer/PSACoverCropCouncil.jsx:76]: Done @YazidZidane

**Review by YazidZidane — APPROVED** [review]

**Pull request merged**